content/docs/0.8.1/concepts.md

```diff
@@ -27,36 +31,46 @@ Longhorn is a thin-provisioned storage system. That means a Longhorn volume will
 
 If the users want to clean up the latest snapshot, they can create a new snapshot, then remove the previous "latest" snapshot. 
 
-### Maintenance Mode
+#### Note about Block Level Snapshots
 
-After v0.6.0, when the user attaching the volume from Longhorn UI, there is a checkbox for `Maintenance mode`. The option will result in attaching the volume without enabling the frontend (block device or iSCSI), to make sure no one can access the volume data when the volume is attached.
+Longhorn is a `crash-consistent` block storage solution.
 
-It's mainly used to perform `Snapshot Revert`. After v0.6.0, Snapshot Reverting operation required volume to be in `Maintenance mode` since we cannot modify the block device's content with the volume mounted or being used, otherwise it will cause filesystem corruptions. 
+It's normal for the OS to keep content in the cache before writing into the block layer. However, it also means if the all the replicas are down, then Longhorn may not contain the immediate change before the shutdown, since the content was kept in the OS level cache and hadn't transferred to Longhorn system yet. It's similar to if your desktop was down due to a power outage, after resuming the power, you may find some weird files in the hard drive.
 
-It's also useful to inspect the volume state without worry that the data can be accessed by accident.
+To force the data being written to the block layer at any given moment, the user can run `sync` command on the node manually, or unmount the disk. OS would write the content from the cache to the block layer in either situation.
 
-### Note about Block Level Snapshots
+### Backups
 
-Longhorn is a `crash-consistent` block storage solution.
+A backup in Longhorn represents a volume state (a Snapshot) at a given time, stored in the secondary storage (the Longhorn backupstore) which is outside of the Longhorn system. Backup creation will involving copying the data through the network, so it will take time.
 
-It's normal for the OS to keep content in the cache before writing into the block layer. However, it also means if the all the replicas are down, then Longhorn may not contain the immediate change before the shutdown, since the content was kept in the OS level cache and hadn't transferred to Longhorn system yet. It's similar to if your desktop was down due to a power outage, after resuming the power, you may find some weird files in the hard drive.
+These operations happen automatically, but can also be done as needed.
```

Comment:
`Backup operation can be scheduled using the recurring snapshot and backup feature, but it can also be done as needed. It's recommended to schedule recurring backups for your volumes. If a backupstore is not available, it's recommended to have the recurring snapshot scheduled instead.`

Reply:
added sentences